res/values-tl/strings.xml

```diff
@@ -141,17 +141,17 @@
     <string name="Blobihilation">"Blobihilation:"</string>
     <string name="Blobageddon">"Blobageddon:"</string>
     <string name="Blobocalypse">"Blobocalypse:"</string>
-    <string name="Experienced_Nebulator">"Experienced Nebulator:"</string>
-    <string name="Expert_Nebulator">"Expert Nebulator:"</string>
+    <string name="Experienced_Nebulator">"Nakaranas Ng Nebulator:"</string>
+    <string name="Expert_Nebulator">"Dalubhasang Nebulator:"</string>
     <string name="Master_Nebulator">"Master Nebulator:"</string>
     <string name="Grand_Master_Nebulator">"Grand Master Nebulator:"</string>
     <string name="Impossible">"Imposible:"</string>
     <string name="Perfection">"Perpekto:"</string>
-    <string name="Single_Handedly">"Single-handedly:"</string>
+    <string name="Single_Handedly">"Single Handedly:"</string>
     <string name="Dodgey">"Dodgey:"</string>
-    <string name="Self_Contained">"Self-Contained:"</string>
+    <string name="Self_Contained">"Self_Contained:"</string>
```

Comment:
Please remove the bottom line and leave a space instead. You don't need to base off completely on the english value (blue colour texts).